src/overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -275,18 +328,24 @@ def execute_plan(self, start_state, joint_action_plan, display=False):
             if display: print(self)
             if done: break
         successor_state = self.state
-        self.reset()
+        self.reset(False)
         return successor_state, done
 
-    def run_agents(self, agent_pair, include_final_state=False, display=False, display_until=np.Inf):
+    def run_agents(self, agent_pair, include_final_state=False, display=False, dir=None, display_until=np.Inf):
         """
         Trajectory returned will a list of state-action pairs (s_t, joint_a_t, r_t, done_t, info_t).
         """
         assert self.state.timestep == 0, "Did not reset environment before running agents"
         trajectory = []
         done = False
-
-        if display: print(self)
+        # default is to not print to file
+        fname = None
+
+        if dir != None:
+            fname = dir + '/roll_out_' + str(time.time()) + '.txt'
+            f = open(fname, 'w+')
+            print(self, file=f)
+            f.close()
```

Comment:
Now dir is passed down as an argument from evaluation_params. @micahcarroll  please let me know how this solution looks to you.